ref(sveltekit): Update SvelteKit SDK compatibility note (#7073)
getsentry/sentry-docs: src/platforms/javascript/guides/sveltekit/manual-setup.mdx

};
```

Using the `sourceMapsUploadOptions` object is useful if the default source maps upload doesn't work out of the box, for instance, if you have a customized build setup or if you're using the SDK with a SvelteKit adapter other than the [supported adapters](../#compatibility).

### Overriding SvelteKit Adapter detection

> By default, `sentrySvelteKit` will try to detect the used SvelteKit adapter to configure source maps upload correctly for the respective adapter.

Reviewer: ```suggestion
By default, `sentrySvelteKit` will try to detect your SvelteKit adapter to configure source maps upload correctly.
```

> If we detect the wrong adapter our you're using another adapter than the [supported adapters](../#compatibility), you can override the adapter detection by passing the `adapter` option to `sentrySvelteKit`:

Reviewer: ```suggestion
If you're not using one of the [supported adapters](../#compatibility) or the wrong one is detected, you can override the adapter detection by passing the `adapter` option to `sentrySvelteKit`:
```
